### PR TITLE
Adding policy examples for hostpath mounts and CVE-2021-43816

### PR DESCRIPTION
**Pull request by sudermanjr**
## Description
Adds two policy examples that are related to CVE-2021-43816

- Just alerting on any volume that has a hostPath
- Alerting if there is a hostPath volume AND a volumeMount in one of the locations specified in CVE-2021-43816

## Checklist

* [x] I have updated the CHANGELOG of any plugins updated
* [x] I have updated the version file of any plugins updated
* [x] I have added tests wherever appropriate
* [x] I have labeled this PR with a label per plugin updated

### CODE REVIEW

**Comment by sudermanjr** [comment]
Screen Shot 2022-01-05 at 5 34 55 PM

Example action item showing a test exploitation of the CVE

File: plugins/opa/examples/containerd-hostpath-cve-2021-43816/policy.rego [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        input.kind == "CronJob"
+        pod := input.spec.jobTemplate.spec.template
+    podHasHostPathVolume(pod)
+        podHasDangerousVolumeMount(pod)
+        actionItem := {
+                "title": concat(" ", [input.kind, " may be exploiting CVE-2021-43816"]),
+                "description": "Pod has a hostPath mount and a volumeMount at a vulnerable path. It may be exploiting CVE-2021-43816",
+                "remediation": "Immediately investigate this workload and mitigate CVE-2021-43816",
+                "category": "Security",
+        "severity": .99
```
Comment (rbren) [inline plugins/opa/examples/containerd-hostpath-cve-2021-43816/policy.rego:30]: indent is off in a few places

File: plugins/opa/examples/containerd-hostpath-cve-2021-43816/policy.rego [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+checkPod[actionItem] {
+        input.kind == "Pod"
+
+        # Only alert for stand-alone pods,
+        # avoiding duplicate action-items for pods which belong to a controller.
+        not input.metadata.ownerReferences
+    podHasHostPathVolume(input)
+        podHasDangerousVolumeMount(input)
+        actionItem := {
```
Comment (rbren) [inline plugins/opa/examples/containerd-hostpath-cve-2021-43816/policy.rego:59]: I'm sure there's a way to extract out the common stuff once you have the pod spec. But I don't really know what it is off hand.

File: plugins/opa/examples/containerd-hostpath-cve-2021-43816/policy.rego [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+checkPod[actionItem] {
+        input.kind == "Pod"
+
+        # Only alert for stand-alone pods,
+        # avoiding duplicate action-items for pods which belong to a controller.
+        not input.metadata.ownerReferences
+    podHasHostPathVolume(input)
+        podHasDangerousVolumeMount(input)
```
Comment (rbren) [inline plugins/opa/examples/containerd-hostpath-cve-2021-43816/policy.rego:58]: just to be sure - a pod needs BOTH the host path volume AND the dangerous volume mount to be vulnerable, right?

**Review by rbren — APPROVED** [review]

File: plugins/opa/examples/containerd-hostpath-cve-2021-43816/policy.rego (hunk 3 above)
Comment (sudermanjr) [reply plugins/opa/examples/containerd-hostpath-cve-2021-43816/policy.rego:58]: Yes

**Pull request merged**